src/main/java/com/bart/scorebetlive442/entity/TeamEntity.java

```diff
@@ -0,0 +1,54 @@
+package com.bart.scorebetlive442.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+@Table(name = "Teams", indexes =
```

Comment:
pilnujmy takich rzeczy, jak już nasze tabele korzystają  z liczby pojedyn. to tego się trzymajmy (stosowanie pojedynczej formy jest bardziej popularne)